cfy_manager/components/restservice/config/authorization.conf

```diff
@@ -306,6 +306,9 @@ permissions:
   manager_update_fs_sync_node_id:
     - sys_admin
 
+  broker_get:
+    - sys_admin
```

Comment:
Shouldn't this be accessible by default, otherwise agent installs will fail for non-admin users?